sdk/core/azure-core/azure/core/pipeline/transport/_aiohttp.py

```diff
@@ -145,6 +150,10 @@ async def send(self, request: HttpRequest, **config: Any) -> Optional[AsyncHttpR
         :keyword str proxy: will define the proxy to use all the time
         """
         await self.open()
+        try:
+            auto_decompress = self.session.auto_decompress  # type: ignore
+        except AttributeError:
+            auto_decompress = True
```

Comment:
I would add a comment as to why we need this. I know I would be confused unless I knew the history...

Reply:
Added.